test/integration/scheduler_test.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/kubernetes/pkg/scheduler"
+	schedapi "k8s.io/kubernetes/pkg/scheduler/apis/config"
+	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
+	testutils "k8s.io/kubernetes/test/integration/util"
+
+	"sigs.k8s.io/scheduler-plugins/pkg/qos"
+)
+
+func TestSchedulerCreation(t *testing.T) {
+	tests := []struct {
+		name            string
+		registry        framework.Registry
+		plugins         *schedapi.Plugins
+		expectedPlugins map[string][]schedapi.Plugin
+	}{
+		{
+			name:     "QoS QueueSort plugin",
+			registry: framework.Registry{qos.Name: qos.New},
+			plugins: &schedapi.Plugins{
+				QueueSort: &schedapi.PluginSet{
+					Enabled: []schedapi.Plugin{
+						{Name: qos.Name},
+					},
+					Disabled: []schedapi.Plugin{
+						{Name: "*"},
+					},
+				},
+			},
+			expectedPlugins: map[string][]schedapi.Plugin{
```

Comment:
is this test passing? I think we should be including the rest of default plugins here.

Reply:
I added some specific logic below to only check the specified out-of-tree entries are present.